sdk/KoreBuild/modules/frameworkbuild/4-package.targets

```diff
@@ -0,0 +1,64 @@
+<Project>
+  <PropertyGroup>
+    <PackageDependsOn>$(PackageDependsOn);PackageRuntimeStore;PackageMetaPackage</PackageDependsOn>
```

Comment:
Again, not sure how these package steps are supposed to be invoked. The PackageRuntimeStore target will need to be run 4 times whereas PackageMetaPackage only need to be run once. They don't belong in the same list of targets.